src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private Random random = new Random();
```

Comment:
ColorSupplier and random are not a part of Figure. 

Shouldn't be here. 
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();

    private Random random = new Random();
```